Boundary Docs (#2925)
hashicorp/web-unified-docs: content/boundary/v1.0.x/content/docs/targets/sessions/cancel-sessions.mdx

---
layout: docs
page_title: Cancel a Boundary session

> description: Manually cancel a Boundary session to free resources or stop unauthorized access, via the Admin Console or the CLI.

Reviewer: ```suggestion
description: Manually cancel a Boundary session to free resources or stop unauthorized access, via the Admin UI or the CLI.
```

---

> # Cancel a Boundary session

Reviewer: ```suggestion
# Cancel a session
```

> You can manually cancel a Boundary session to free resources or stop unauthorized access — for example, if you detect unexpected activity — using the Admin Console or the boundary sessions cancel CLI command. Boundary also cancels sessions automatically when they reach their time limit or a related resource is removed.

Reviewer: ```suggestion
You can manually cancel a Boundary session using the Admin UI or the boundary sessions cancel CLI command to free resources or stop unauthorized access, for example, if you detect unexpected activity. Boundary cancels sessions automatically when they reach their time limit or a related resource is removed.
```
